app/controllers/manifest_uploads_controller.rb

```diff
@@ -1,5 +1,46 @@
 class ManifestUploadsController < ApplicationController
   def new
+  end
+
+  def create
+    manifest = find_or_initialize_manifest
+
+    if manifest.save
+      flash[:notice] = "Upload for manifest #{manifest.tracking_number} submitted successfully."
+      redirect_to root_path
+    end
+  end
+
+  private
+
+  def find_or_initialize_manifest
+    if params[:manifest_id]
+      manifest = Manifest.find(params[:manifest_id])
+      manifest.content[:image] = image_details
```

Comment:
good idea, will change key to `uplaoded_file`

in terms of "one image" - that is my understanding of the current needs. also confirmed here that there is only one manifest per image file: https://trello.com/c/djblE1XG